python_xbcf/xbcausalforest/xbcf_python.py

```diff
@@ -452,17 +489,41 @@ def predict(self, x_test, return_mean=True):
         )
 
         # Convert from colum major
-        self.tauhats_test = tauhats_test.reshape(
```

Comment:
suggestion: don't include prediction output into model object to avoid confusion if predict is called on different data and to adhere closer to scikit-learn convention